src/software/math/math_functions.h

```diff
@@ -94,6 +94,22 @@ T normalizeValueToRange(T value, const T& value_min, const T& value_max,
     return new_range / value_range * (value - value_max) + range_max;
 }
 
+/**
+ * Returns the sign (-1, 0, or 1) of a number
+ *
+ * @tparam T The type of the given number. Must be an integral or floating point type
+ * @param value The number to return the sign of
+ *
+ * @return The sign (-1, 0, or 1) of the number
+ */
+template <typename T>
+int signum(T value)
+{
+    static_assert(std::is_integral<T>::value || std::is_floating_point<T>::value,
+                  "Integral or floating point type required.");
+    return (T(0) < value) - (value < T(0));
+}
```

Comment:
can we use `std::copysign` (https://en.cppreference.com/w/cpp/numeric/math/copysign)

Reply:
removed